frontend/src/components/MobileFilterMenu/MobileFilterSubMenu.tsx

```diff
@@ -105,16 +105,45 @@ export const MobileFilterSubMenu: React.FC<Props> = ({
             hideLabel
           />
         ))}
-
+        {entriesFilters[0][1].id === EVENT_ID && (
+          <div className="flex flex-col mt-4 desktop:mt-0 desktop:ml-5">
+            <div className="font-bold mb-2 text-lg">Agenda</div>
+            <InputDateWithMagnifier
+              value={dateFilter.beginDate}
+              onChange={event => {
+                setDateFilter({
+                  beginDate: event.target.value,
+                  endDate: dateFilter.endDate,
+                });
+              }}
+              placeholder={intl.formatMessage({ id: 'search.beginDateFilter' })}
+            />
+            <InputDateWithMagnifier
+              value={dateFilter.endDate}
+              onChange={event => {
+                setDateFilter({
+                  beginDate: dateFilter.beginDate,
+                  endDate: event.target.value,
+                });
+              }}
+              placeholder={intl.formatMessage({ id: 'search.endDateFilter' })}
+            />
+          </div>
+        )}
```

Comment:
This condition looks like a patch.
Perhaps it would be better to leave this loop alone and move this condition into the `<ShowFilter />` component (or a new intermediate component) and display a date range instead of a select/filter button?